src/test/java/core/basesyntax/service/impl/DataParserImplTest.java

```diff
@@ -0,0 +1,52 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.DataParser;
+import core.basesyntax.service.impl.exception.InvalidDataException;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class DataParserImplTest {
+    private static DataParser dataParser;
+    private static FruitTransaction bananaTransaction;
```

Comment:
Better to create this transactions in getParsedData_validParsedData_Ok method, they used only once